src/runtime_src/core/tools/xbutil2/SubCmdAdvanced.cpp

```diff
@@ -78,6 +79,7 @@ SubCmdAdvanced::execute(const SubCmdOptions& _options) const
 // Only defined for embedded platform
 #ifndef ENABLE_NATIVE_SUBCMDS_AND_REPORTS
   subOptionOptions.emplace_back(std::make_shared<OO_AieRegRead>("read-aie-reg"));
```

Comment:
I would be better if the `read-aie-reg` option be changed to `aie-reg` with a `read `option.   Doing so allows for this operation to be scaled in the future.   

Note: Being that this is in the ADVANCE command, we can change these options without having to consult with the PRT.

Reply:
@rozumx I will try to address this in separate PR as this requires document change as well